kolibri/plugins/learn/assets/src/composables/useDownloadRequests.js

```diff
@@ -27,107 +31,75 @@ const downloadRequestMap = reactive({
   downloads: {},
   totalPageNumber: 0,
   totalDownloads: 0,
+  totalStorage: 0,
```

Comment:
I.. don't think I did this in my original implementation - but this looks like a bad idea.

We should keep `downloadRequestMap` as just a map of downloads (so the `downloads` key should be the top level object).

We should then move each of these other keys into a separate ref value.